catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlQueryResponse.java

```diff
@@ -57,6 +59,14 @@ public CqlQueryResponse(String id, QueryRequest request, QueryResponse queryResp
 
         status = new Status(queryResponse, source, elapsedTime);
 
+        AtomicBoolean logOnceState = new AtomicBoolean(false);
+        Consumer<String> logOnce = (str) -> {
```

Comment:
✏️ Not in the scope of this change, but this is the sort of functionality that would be nice to add to a DDF logger service/utility. Something to wrap (and hide) the underlying logger implementation we use and decorate with enhanced functionality.